tsconfig.paths.json

```diff
@@ -0,0 +1,10 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "types": [
+        "../head-hunter-backend/src/types"
```

Comment:
this is important information.  Name of backend folder must be the same for everyone.

Reply:
this is root name from git clone,  important to have front-end and back-end in the same directory  and oryginal names from git clone